index.js

```diff
@@ -96,43 +96,79 @@ original error: ${e}`);
   }
 }
 
-async function exportPodcasts(podcastsDBData) {
+
+async function buildPodcastDict(fileName, cacheFilesPath, podcastsDBData) {
+  const uuid = fileName.replace(".mp3", "");
+  const dbMeta = podcastsDBData.find((m) => m.zuuid === uuid);
+  const path = `${cacheFilesPath}/${fileName}`;
+  const exportBase = dbMeta?.zcleanedtitle // 1. from apple podcast database
+        ?? (await getMP3MetaTitle(path)) // 2. from mp3 meta data
+        ?? uuid; // 3. fallback to unreadable uuid
+  const podcastName = dbMeta?.zpodcast.replaceAll('/', '_');
+  const exportFileName = sanitize(exportBase.substr(0, fileNameMaxLength));
+  const date = dbMeta?.date
+
+  const ret = {
+    podcastName,
+    date,
+    fileName,
+    path,
+    uuid,
+    exportFileName: `${exportFileName}.mp3`
+  };
+  return ret;
```

Comment:
probably just return looks cleaner
```suggestion
  return {
    podcastName,
    date,
    fileName,
    path,
    uuid,
    exportFileName: `${exportFileName}.mp3`
  };
```

Reply:
Right, I had made a var so I could console.log it, but returning it is fine. :-)

Reply:
Changed.